.github/workflows/docker-push.yaml

```diff
@@ -0,0 +1,27 @@
+---
+name: push
+on:
+  push:
+    branches:
+      - master
+jobs:
+  pushImage:
+    name: push image
+    runs-on: ubuntu-latest
+    env:
+      DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
```

Comment:
@edwarnicke Can you please add these envs?

Reply:
Done

Reply:
Cool, the job will trigger on merge into master :)